corehq/apps/cloudcare/templates/formplayer/grid_view.html

```diff
@@ -111,7 +111,7 @@ <h3>{% trans "Settings" %}</h3>
   <div class="container">
     <div class="spacer"></div>
     <div class="col-xs-6 col-sm-4 col-xs-offset-3 col-sm-offset-4">
-      <div class="appicon appicon-default js-start-app">
+      <div class="appicon appicon-default js-start-app" role="link" tabindex="0" aria-label="<%= appName %>">
```

Comment:
The app name should use `<%- ... %>` so it gets escaped. We just did a bunch of escaping fixes in https://github.com/dimagi/commcare-hq/pull/29471

Reply:
OK, done.